scripts/build-layout-scales.js

```diff
@@ -0,0 +1,1271 @@
+          padding-left: ${value(scale)} !important;
+        }
+      `);
+    }, '');
+
+    const medium = scales.reduce((result, scale) => {
+      return result += stripIndent(`
+        .grid--gut${scale}-mm {
+          margin-left: -${value(scale)} !important;
+        }
+      `);
+    }, '');
+
+    const mediumColumns = scales.reduce((result, scale) => {
+      return result += stripIndent(`
+        .grid--gut${scale}-mm > .col {
+          padding-left: ${value(scale)} !important;
+        }
+      `);
+    }, '');
+
+    css += stripIndent(`
+      @media ${mediaQueries['--m-screen']} {
+        ${medium}
+        ${mediumColumns}
+      }
+    `);
+
+    const large = scales.reduce((result, scale) => {
+      return result += stripIndent(`
+        .grid--gut${scale}-ml {
+          margin-left: -${value(scale)} !important;
+        }
+      `);
+    }, '');
+
+    const largeColumns = scales.reduce((result, scale) => {
+      return result += stripIndent(`
+        .grid--gut${scale}-ml > .col {
+          padding-left: ${value(scale)} !important;
+        }
+      `);
+    }, '');
+
+    css += stripIndent(`
+      @media ${mediaQueries['--l-screen']} {
+        ${large}
+        ${largeColumns}
+      }
+    `);
+
+    const xlarge = scales.reduce((result, scale) => {
+      return result += stripIndent(`
+        .grid--gut${scale}-mxl {
+          margin-left: -${value(scale)} !important;
+        }
+      `);
+    }, '');
+
+    const xlargeColumns = scales.reduce((result, scale) => {
+      return result += stripIndent(`
+        .grid--gut${scale}-mxl > .col {
+          padding-left: ${value(scale)} !important;
+        }
+      `);
+    }, '');
+
+    css += stripIndent(`
+      @media ${mediaQueries['--xl-screen']} {
+        ${xlarge}
+        ${xlargeColumns}
+      }
+    `);
+
+    return css;
+  };
+
+  variantGenerators.zIndex = function (scales) {
+    function value(v) {
+      return typeof v === 'string' ? `-${v[v.length - 1]}` : v;
+    }
+
+    let css = stripIndent(`
+    /**
+     * Override default stacking order by setting an element's <code>z-index</code> property value. Whenever possible, rely on
+     * built-in stacking order and avoid using these classes. Class set includes <code>*-mm</code>, <code>*-ml</code>, and <code>*-mxl</code> variations to target screen sizes.
+     *
+     * @group
+     * @memberof Positioning
+     * @example
+     * <div class="relative h72">
+     *   <div class="absolute z5 bg-pink p6">z5</div>
+     *   <div class="absolute z1 bg-blue p6 mt12 ml24">z1</div>
+     *   <div class="absolute z1-neg1 bg-teal p6 mt24 ml48">z-neg1</div>
+     * </div>
+     */`
+    );
+    css += scales.reduce((result, scale) => {
+      return result += stripIndent(`
+        .z${scale} {
+          z-index: ${value(scale)} !important;
+        }
+      `);
+    }, '');
+    css += '/** @endgroup */\n';
+
+    css += stripIndent(`\n\n@media ${mediaQueries['--m-screen']} {`);
```

Comment:
No need for `stripIndent` on these single-line strings that have no indent to strip.